test/e2e/images_test.go

```diff
@@ -199,33 +191,23 @@ WORKDIR /test
 	})
 
 	It("podman images filter after image", func() {
-		podmanTest.RestoreAllArtifacts()
-		rmi := podmanTest.PodmanNoCache([]string{"rmi", "busybox"})
-		rmi.WaitWithDefaultTimeout()
-		Expect(rmi).Should(Exit(0))
-
 		dockerfile := `FROM quay.io/libpod/alpine:latest
 `
 		podmanTest.BuildImage(dockerfile, "foobar.com/before:latest", "false")
-		result := podmanTest.PodmanNoCache([]string{"images", "-q", "-f", "after=quay.io/libpod/alpine:latest"})
+		result := podmanTest.Podman([]string{"images", "-q", "-f", "after=quay.io/libpod/alpine:latest"})
 		result.WaitWithDefaultTimeout()
 		Expect(result).Should(Exit(0))
-		Expect(len(result.OutputToStringArray())).To(Equal(0))
+		Expect(len(result.OutputToStringArray())).To(Equal(7))
 	})
 
 	It("podman image list filter after image", func() {
-		podmanTest.RestoreAllArtifacts()
-		rmi := podmanTest.PodmanNoCache([]string{"image", "rm", "busybox"})
-		rmi.WaitWithDefaultTimeout()
-		Expect(rmi).Should(Exit(0))
-
 		dockerfile := `FROM quay.io/libpod/alpine:latest
 `
 		podmanTest.BuildImage(dockerfile, "foobar.com/before:latest", "false")
-		result := podmanTest.PodmanNoCache([]string{"image", "list", "-q", "-f", "after=quay.io/libpod/alpine:latest"})
+		result := podmanTest.Podman([]string{"image", "list", "-q", "-f", "after=quay.io/libpod/alpine:latest"})
```

Comment:
This test is no longer checking after.

We need to add an image, and then test after it and make sure we don't get any images.